Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -957,6 +959,17 @@ class TrackStateProxy {
 
   const MultiTrajectory<Trajectory>& trajectory() const { return *m_traj; }
 
+  /// Get a mutable reference to the track state container backend
+  /// @return a mutable reference to the backend
+  template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
+  auto& container() {
```

Comment:
Not sure this is "future" safe, we give access to underying backend which shold not be a public API. No?

Reply:
I don't necessarily think it's dangerous to expose the backends like this. Do you think this would cause problems? The track container I believe already does this now

Reply:
I actually noticed that this is already accessible before, I duplicated these functions. I'll synchronize.